frontend/src/components/dialogs/CreateFolderDialog.tsx

```
@@ -0,0 +1,195 @@
+                className="w-full px-4 py-2.5 bg-gray-50 dark:bg-gray-800 border border-gray-200 dark:border-gray-700 rounded-xl text-sm focus:ring-2 focus:ring-purple-500 focus:border-transparent outline-none transition-all placeholder-gray-400" 
+                placeholder="e.g., Design Resources"
+                value={folderName}
+                onChange={(e) => setFolderName(e.target.value)}
+              />
+            </div>
+
+            {/* 2. 외형 설정 (아이콘 선택) 섹션 */}
+            <div>
+              <div className="flex items-center gap-2 mb-3">
+                <span className="w-1.5 h-1.5 rounded-full bg-purple-500"></span>
+                <label className="text-sm font-semibold text-gray-700 dark:text-gray-200">Appearance</label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)